docs/core/build/distribution-packaging.md

```diff
@@ -16,7 +16,7 @@ As .NET Core becomes available on more and more platforms, it's useful to learn
 
 When installed, .NET Core consists of several components that are laid out as follows in the filesystem:
 
-```
+```console
```

Comment:
This one is hard. Maybe we should skip it? It's just showing the file structure in a visual way.

```suggestion
```
```